apps/api/app/Domain/Bank/Actions/ComputeBankProvisions.php

```
@@ -0,0 +1,284 @@
+        Collection $collected,
+        Collection $movements,
+        CarbonImmutable $today,
+        string $currency,
+    ): BankProvisionData {
+        $rateBp = $settings->effectiveContributionRateBp();
+
+        $current = $this->contributionsAccruedBetween($collected, $period['start'], $today, $rateBp, $currency);
+        $carried = max(
+            0,
+            $this->contributionsAccruedBetween($collected, $period['previousStart'], $period['start']->subDay(), $rateBp, $currency)
+                - $this->paymentsBetween($movements, $period['start'], $today, DetectFiscPayments::isUrssaf(...)),
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'App\Domain\Bank\Actions\DetectFiscPayments' in method 'urssaf'. (undefined)

(StaticAccess)